indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -62,22 +62,26 @@ def _process(self):
 
 class RHSearchRooms(RHRoomBookingBase):
     @use_args(dict(search_room_args, **{
-        'unavailable': fields.Bool(missing=False)
+        'unavailable': fields.Bool(missing=False),
+        'admin_override_enabled': fields.Bool()
```

Comment:
is it always in the request or should we be on the safe side and add `missing` to the `fields.Bool`?

Reply:
missing defaults to `Missing`, which is falsy

Reply:
Again here, if there was a missing option that would be False, correct?

Reply:
it'd be a falsy object. if you specify `missing=False` it will be actually `False`